ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -1330,6 +1339,46 @@ ORK_CLASS_AVAILABLE
 
 
 /**
+ The `ORKHeightAnswerFormat` class represents the answer format for questions that require users
+ to enter a height.
+ 
+ A date answer format produces an `ORKNumericQuestionResult` object. The result unit will either be
+ `cm` or `in` depending on the measuring system chosen when building the answer format.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKHeightAnswerFormat : ORKAnswerFormat
+
+/**
+ Returns an initialized height answer format using the measuring system specified in the current
+ locale.
+ 
+ @return An initialized height answer format.
+ */
+- (instancetype)init;
+
+/**
+ Returns an initialized height answer format using the specified meauring system.
+ 
+ This method is the designated initializer.
+ 
+ @param useMetricSystem     The measuring system to use. Set it to `YES` to use the metric system,
+ set it to `NO` to use the imperial system.
+ 
+ @return An initialized height answer format.
+ */
+- (instancetype)initWithMetricSystem:(BOOL)useMetricSystem NS_DESIGNATED_INITIALIZER;
+
+/**
+ Indicates the measuring system used by the answer format.
+ 
+ A value of `NO` indicates that the imperial system is used.
+ */
+@property (readonly) BOOL useMetricSystem;
```

Comment:
If you want that all results are reported in metric regardless of the `useMetricSystem` value, I can certainly do that.